### PR TITLE
fix: return the right month in getStartOfDate

### PR DESCRIPTION
**Pull request by andresmoschini**
Fix this:

image

Notice that the month is wrong. More details: https://makingsense.slack.com/archives/CGJ72QFQX/p1572470224155300

I will merge without review.

### CODE REVIEW

**Comment by andresmoschini** [comment]
Tested (minimally) in https://cdn.fromdoppler.com/doppler-webapp/demo-build1787/#/reports

File: src/components/Reports/Reports.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { addDays } from '../../utils';
 const periodSelectedDaysDefault = 7;
 
 const getStartOfDate = (date) => {
-  return new Date(date.getFullYear(), date.getMonth() + 1, date.getDate());
+  return new Date(date.getFullYear(), date.getMonth(), date.getDate());
```
Comment (cbernat) [inline src/components/Reports/Reports.js]: 👍

**Comment by fcoronelmakingsense** [comment]
Thanks very much for the fix @andresmoschini

**Pull request merged**